hw/xbox/nv2a/nv2a_vsh.c

```diff
@@ -627,7 +627,16 @@ static const char* vsh_header =
     "#define ARL(dest, src) dest = _ARL(_in(src).x)\n"
     "int _ARL(float src)\n"
     "{\n"
-    "  return int(floor(src));\n"
+    "  /* Xbox GPU does specify rounding, OpenGL doesn't; so we need a bias.\n"
+    "   * Example: We probably want to floor 16.99.. to 17, not 16.\n"
+    "   * Source of error (why we get 16.99.. instead of 17.0) is typically\n"
```

Comment:
nit: "typically" &rarr; "probably"

There's some uncertainty in #78 regarding why this issue doesn't also affect a real Xbox.